packages/components-js/src/pages/popover.html

```diff
@@ -1,4 +1,8 @@
-<div class="playground light" title="should render multiple popovers" style="position: relative; height: 500px">
+<div
+  class="playground light"
+  title="should render multiple popovers on light background"
+  style="position: relative; height: 500px"
+>
```

Comment:
Safari screenshot looks like this?
Besides the missing backgrounds on light theme, I would suggest adding a link, bold and italic text on dark theme, too.

<img width="438" alt="Bildschirm­foto 2023-01-26 um 07 55 24" src="https://user-images.githubusercontent.com/4380295/214775133-4b2a1f00-4e4a-4c18-be23-e6b6f4253981.png">

Reply:
In noticed that the safari screenshot looks like this, but it did so before. 
As it is supported in Safari (See screenshot) i did not investigate further.

<img width="629" alt="Bildschirmfoto 2023-01-26 um 08 53 26" src="https://user-images.githubusercontent.com/60395476/214783784-f17f3e19-54d7-4630-933f-01139428a5dd.png">

Reply:
Ill add link, bold and italic text on dark theme too